medicines/web/src/components/search-results/pagination.tsx

```diff
@@ -65,58 +65,54 @@ const StyledPagination = styled.nav`
   }
 `;
 
-const Pagination = (props: {
+interface IPaginationProps {
   searchTerm: string;
   resultCount: number;
   pageSize: number;
   currentPage: number;
   enabledDocTypes: DocType[];
-}) => {
+  handlePageChange: (num: number) => void;
+}
+
+const Pagination = (props: IPaginationProps) => {
   const pageCount = Math.floor(props.resultCount / props.pageSize) + 1;
   const { firstGroup, middleGroup, lastGroup } = getPaginationGroups(
     pageCount,
     props.currentPage,
   );
 
+  const getHandlePageChangeHandler = (num: number) => () =>
+    props.handlePageChange(num);
+
   const createPaginationButton = (page: number, i: number, array: number[]) => {
     const separator = i === array.length - 1 ? '' : <span>&ndash;</span>;
 
     if (page === props.currentPage) {
       return (
-        <li key={page}>
+        <li key={page + i}>
```

Comment:
Is the `key` now `2 * page - 1`? Why?

Reply:
Was playing with the keys to silence the console error around unique keys as was trying to figure out if there were duplicate page number or something but it was actually complaining about the <li> below. Reverted this.